bundle.yaml

```diff
@@ -1,23 +1,23 @@
 series: xenial
 applications: 
   postgresql: 
-    charm: "cs:postgresql-105"
+    charm: "cs:postgresql"
     num_units: 1
     annotations: 
       "gui-x": 1407
       "gui-y": 295.0000305175781
     to: 
       - "0"
   "serial-vault": 
-    charm: "cs:~jamesj/serial-vault-6"
```

Comment:
I'm confused (maybe not using juju in some months has something to do). I see that https://jujucharms.com/u/jamesj/serial-vault says the last version is the 9th. Same with haproxy, where says last is #41